$OpenBSD$

Use GNU od(1) or else converted XML file will have invalid digits

Index: Build/Include/Makefile.inc
--- Build/Include/Makefile.inc.orig
+++ Build/Include/Makefile.inc
@@ -47,7 +47,11 @@ endif
 
 
 # Embedded files
+ifeq "$(PLATFORM)" "OpenBSD"
+OD_BIN := ggod -v -t u1 -A n
+else
 OD_BIN := od -v -t u1 -A n
+endif
 TR_SED_BIN := tr '\n' ' ' | tr -s ' ' ',' | sed -e 's/^,//g' -e 's/,$$/n/' | tr 'n' '\n'
 
 %.xml.h: %.xml
